src/main/java/com/avispl/symphony/dal/communicator/middleatlantic/SocketCommunicator.java

```diff
@@ -0,0 +1,460 @@
+package com.avispl.symphony.dal.communicator.middleatlantic;
```

Comment:
No copyrights? Can you please add a template for new files to you IDE to generate copyrights header for Java files automatically

Reply:
Sure, adding them. I'll also check if there's a way to add this into the PMD settings so the project won't compile unless copyrigths are present.